catalog/transformer/catalog-transformer-zip/pom.xml

```diff
@@ -65,11 +75,27 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.name}</Bundle-Name>
                         <Embed-Dependency>
+                            ant,
+                            ant-launcher,
                             catalog-core-api-impl,
                             platform-util,
                             zip4j
                         </Embed-Dependency>
-
+                        <Import-Package>
+                            !com.sun.tools.apt,
```

Comment:
Needing to stop things from being imported is generally a smell. It seems like these would be better served as excluded dependencies, which should also keep OWASP from complaining if one of them has a CVE.